content/gettingStarted/examples/multiPlayerEnv/_index.en.md

```diff
@@ -15,4 +15,14 @@ This example focuses on:
 A dedicated section describing environment settings is presented <a href="../../../envs/#settings">here</a>.
 {{% /notice %}}
 
-{{< github_code "https://raw.githubusercontent.com/diambra/arena/main/examples/multi_player_env.py" >}}
\ No newline at end of file
+{{< github_code "https://raw.githubusercontent.com/diambra/arena/main/examples/multi_player_env.py" >}}
+
+{{% notice note %}}
+You can employ a single trained agent to play against itself in a two-player game setting. To do this effectively, you must separate the observations for each player, ensuring they are formatted correctly according to the model's training configuration. Additionally, actions generated by the model for each player need to be handled distinctly to maintain the integrity of the game's mechanics.
+<br><br>
+The following example demonstrates utilizing a single model for self-play within a two-player environment. It emphasizes the importance of correctly segregating observations and actions for both players:
```

Comment:
> players:

remove the : at the end of the sentence and put a . (unless you wanted to add a part that is missing)